api/read

```diff
@@ -41,7 +41,10 @@ else
     channels=$(_query "select count(*) from channels where deleteat = 0")
     posts=$(_query "select count(*) from posts where deleteat = 0")
 
+    database_usage=$(su - postgres -c "scl enable rh-postgresql94 -- psql -q -A -t --port=55432 -c 'SELECT  pg_database_size('\''mattermost'\'');'")
+    data_usage=$(/usr/bin/du -s /var/lib/nethserver/mattermost | cut -f1)
```

Comment:
On first run, this call can take a long time.
On our server:
```
real	0m4.275s
user	0m0.323s
sys	0m0.755s
```
I think it is still acceptable for most system, also I do not think most of machines have such a big directory.